src/com/kinvey/sample/kitchensink/push/PushFragment.java

```diff
@@ -64,16 +67,15 @@ public void onTick(long miliseconds){}
             @Override
             public void onFinish(){
                 //after 5 seconds update the status
-                if (getApplicationContext() != null)
-                pushStatus.setText(Boolean.toString(getApplicationContext().getClient().push().isPushEnabled()));
+                Log.i(Client.TAG, "CountDownTimer - onFinish - " + Boolean.toString(getApplicationContext().getClient().push().isPushEnabled()));
```

Comment:
move this line down below the if statement -- if getApplicationContext is null, then this line will throw a null pointer.